src/databricks/labs/remorph/reconcile/connectors/snowflake.py

```diff
@@ -79,7 +82,10 @@ def get_schema(
             SnowflakeDataSource._SCHEMA_QUERY.format(catalog=catalog, schema=schema, table=table),
         )
         try:
+            logger.debug(f"Fetching schema using query: \n`{schema_query}`")
+            logger.warning(f"Fetching Schema: Started at: {datetime.now()}")
```

Comment:
```suggestion
            logger.info(f"Fetching Schema: Started at: {datetime.now()}")
```
change it to info for both.